### DISCUSSION TITLE
Filter to show a preview of an article

### DISCUSSION THREAD

**Discussion by Obspogon** [11ty/eleventy#2374] [Q&A] [answered]
I'm making a blog and want to show a preview of the post on the home page. How do I get the content of the posts (written in Markdown) and show only part of it?

**Comment by cfjedimaster** [comment]
There's a couple of ways. One is to write a custom filter. You can split the content by paragraphs and return only the first paragraph. You can also return the first X characters. (Here's a good example of that: https://11ty.rocks/eleventyjs/content/#excerpt-filter)

Another option: https://www.11ty.dev/docs/data-frontmatter-customize/.

**Comment by Obspogon** [comment, marked as answer]
Found an answer with [this plugin](https://plug11ty.com/plugins/blog-tools/)
